<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>:/assets/fonts/sub_font.ttf</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Pit Launcher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/assets/images/icon.png</normaloff>:/assets/images/icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: rgb(10, 10, 10);
    color: rgb(52, 0, 222);
	line-height: 20px;
	font-size: 18pt;
}

QPushButton {
    background-color: rgb(10, 10, 10);
    color: rgb(52, 0, 222);
    border: 1px solid rgb(131, 30, 232);
    border-radius: 5px;
    padding: 5px;
}

QPushButton:disabled {
    background-color: rgb(10, 10, 10);
    color: gray;
    border: 1px solid gray;
    border-radius: 5px;
    padding: 5px;
}

QLineEdit {
    background: rgb(10, 10, 10);
    color: rgb(52, 0, 222);
    border: 1px solid rgb(131, 30, 232);
    border-radius: 5px;
    padding: 5px;
}

QComboBox {
    background-color: rgb(10, 10, 10);
    color: rgb(52, 0, 222);
    border: 1px solid rgb(131, 30, 232);
    border-radius: 5px;
}

QCheckBox {
    color: rgb(52, 0, 222);
}

QTabWidget::pane {
    border: 1px solid rgb(131, 30, 232);
}

QTabBar::tab {
    background: rgb(10, 10, 10);
    color: rgb(52, 0, 222);
    border: 1px solid rgb(131, 30, 232);
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
    padding: 5px;
}

QProgressBar {
    color: rgb(255, 255, 255);
    text-align: center;
    font-family: &quot;:/assets/fonts/main_font.ttf&quot;;
    font-size: 28px;
    background: rgb(10, 10, 10);
    border-radius: 5px;
}

QProgressBar::chunk {
    border-radius: 5px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #3400de, stop:1 #831ee8);
}

QComboBox QAbstractItemView {
    background-color: rgb(10, 10, 10);
    selection-background-color: rgb(52, 0, 222);
    selection-color: rgb(10, 10, 10);
    color: rgb(52, 0, 222);
}

QTabBar::tab:selected {
    background: rgb(10, 10, 10);
    color: rgb(52, 0, 222);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0">
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="logoHorizontalLayout">
      <item>
       <widget class="QLabel" name="logo">
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>92</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/assets/images/logo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="username">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: &quot;:/assets/fonts/mian_font.ttf&quot;;
</string>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="start_progress">
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;sub_font&quot;;</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="start_button">
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QTabWidget" name="tab_widget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="vanila_tab">
       <attribute name="title">
        <string>Vanilla</string>
       </attribute>
       <layout class="QVBoxLayout" name="vanila_layout">
        <item>
         <widget class="QComboBox" name="vanilla_combobox"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="forge_tab">
       <attribute name="title">
        <string>Forge</string>
       </attribute>
       <layout class="QVBoxLayout" name="forge_layout">
        <item>
         <widget class="QComboBox" name="forge_combobox"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="fabric_tab">
       <attribute name="title">
        <string>Fabric</string>
       </attribute>
       <layout class="QVBoxLayout" name="fabric_layout">
        <item>
         <widget class="QComboBox" name="fabric_combobox"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="quilt_tab">
       <attribute name="title">
        <string>Quilt</string>
       </attribute>
       <layout class="QVBoxLayout" name="quilt_layout">
        <item>
         <widget class="QComboBox" name="quilt_combobox"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="installed_tab">
       <attribute name="title">
        <string>Installed</string>
       </attribute>
       <layout class="QVBoxLayout" name="installed_layout">
        <item>
         <widget class="QComboBox" name="installed_combobox"/>
        </item>
        <item>
         <widget class="QLineEdit" name="installed_search"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settings_tab">
       <attribute name="title">
        <string>Theme</string>
       </attribute>
       <layout class="QVBoxLayout" name="settings_layout">
        <item>
         <widget class="QComboBox" name="theme_combobox"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
